client/src/app/components/CreateProject/Form.js

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+import ButtonContainer from "../Button/Button";
+import "./styles.css";
+
+class Form extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      titleValue: "",
+      descriptionValue: "",
+      languages: [],
+      complexity: "",
+      members: 0,
+      reset: false
+    };
+  }
+
+  onUpdateSelection(buttonTitle, type) {
+    if (type === "language") {
+      this.setState(prevState => ({
+        languages: [...prevState.languages, buttonTitle]
+      }));
+    } else if (type === "complexity") {
+      this.setState({ complexity: buttonTitle });
+    } else if (type === "members") {
+      this.setState({ members: buttonTitle });
+    }
+  }
+
+  onChangeInput(e) {
+    e.target.name === "project-title"
+      ? this.setState({ titleValue: e.target.value })
+      : this.setState({ descriptionValue: e.target.value });
+  }
+
+  onSubmit(e) {
+    e.preventDefault();
+    console.log(this.state);
+    this.setState(
+      () => ({
+        title: "",
+        description: "",
+        languages: [],
+        complexity: "",
+        members: 0,
+        reset: true
+      }),
+      () => {
+        this.setState({ reset: false });
+      }
+    );
+  }
+
+  render() {
+    return (
+      <div className="form-wrapper">
+        <h2>Create a Project</h2>
+        <div
+          style={{
+            width: "100%",
+            height: "2px",
+            background: "#eee"
+          }}
+        />
+        <form>
+          <div className="title-input-wrapper form-item">
```

Comment:
Optional change, but css-modules can be used to compose style with sth like: 'composes form-item' at the beggining of your '<...>-input-wrapper', which would make the code a bit cleaner

Reply:
Do you want to do that and show me? Do you have time?

Reply:
I won't have much time today, so we can leave it for now. I'll create a separate PR later for it